.appveyor.yml

```diff
@@ -43,7 +43,9 @@ install:
   - set
   - cd c:\tools\vcpkg
   - git pull
-  - git checkout 053f3ee32f309208ac3af9c71b95253d4a144436
+  # - git checkout 053f3ee32f309208ac3af9c71b95253d4a144436
+  - echo.set(VCPKG_BUILD_TYPE release)>> C:\Tools\vcpkg\triplets\%PLATFORM%-windows.cmake
```

Comment:
Can you point at the source of this hint?

Reply:
Tip given here https://github.com/Microsoft/vcpkg/issues/143#issuecomment-346973704 by the maintainer (which was kind enough to commit this  https://github.com/Microsoft/vcpkg/commit/5335d17f53d491d83d98e80ab0e750ec0f6f430b right before ^^)

Reply:
Oh wow, cool!